### DISCUSSION TITLE
install openzeppelin

### DISCUSSION THREAD

**Discussion by parishill25** [Cyfrin/foundry-full-course-cu#1174] [Q&A] [answered]
Hi,
Openzeppelin Upgraded it's libray to version20.I want to install the 18 version what's the command to install the library? thanx in advance

**Comment by 0xusmanf** [comment, marked as answer]
You can specify the version tag like in the example below.
```shell
forge install openzeppelin/openzeppelin-contracts@v4.8.3 --no-commit
```
In this example `v4.8.3` version is installed.
